### PR TITLE
[HackerRank][Tasks] Diagonal Differencer  Task

### PR DESCRIPTION
**Pull request by mirage109**
added readme and solition

### CODE REVIEW

**Comment by KiraSeletska** [comment]
Where I am?

File: HackerRankIssues/Diagonal Differencer/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Given a square matrix, calculate the absolute difference between the sums of its diagonals.
+
+For example, the square matrix  is shown below:
+
+1 2 3
+4 5 6
+9 8 9  
+The left-to-right diagonal = . The right to left diagonal = . Their absolute difference is .
+
+Function description
```
Comment (rasnaut) [inline HackerRankIssues/Diagonal Differencer/README.md:11]: Please Add title style

File: HackerRankIssues/Diagonal Differencer/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+The first line contains a single integer, , the number of rows and columns in the square matrix .
+Each of the next  lines describes a row, , and consists of  space-separated integers .
+
+Constraints
+
+Output Format
+
+Return the absolute difference between the sums of the matrix's two diagonals as a single integer.
+
+Sample Input
```
Comment (rasnaut) [inline HackerRankIssues/Diagonal Differencer/README.md:32]: Please, Highlight the code

**Review by rasnaut — CHANGES_REQUESTED** [review]
Please, make changes according comments

**Pull request merged**